test/api_key_test.rb

```diff
@@ -0,0 +1,127 @@
+# Copyright 2024 Google LLC
```

Comment:
2025

Reply:
Just because it's true doesn't mean I can't be mad about it.